app/controllers/foods_controller.rb

```diff
@@ -1,9 +1,55 @@
 class FoodsController < ApplicationController
-  def index; end
+  def index
+    @foods = Food.where(user_id: current_user.id).includes(:recipe_foods)
```

Comment:
To get all recipe publised by currente_user, you can just implement this :
`@foods =  current_user.foods.includes(:recipe_foods)`